.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+on: [push, pull_request]
```

Comment:
Does this have to be in a hidden directory? Seems likely to make it harder to discover/ grok what's going on.

Reply:
It does, it's a [GitHub convention](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows).